packaging/verify_universal.py

```
@@ -0,0 +1,148 @@
+# regressed to matching nothing (e.g. a `file` output format change, or
+# census()'s rglob/symlink filtering breaking), the fatness gate below would
+# vacuously pass over zero files and sign a thin (or otherwise broken)
+# bundle without ever noticing. This exact "a check that cannot fail"
+# pattern has already caused several defects on this branch.
+MIN_PLAUSIBLE_MACHO_COUNT = 50
+
+
+def run(cmd, **kwargs) -> subprocess.CompletedProcess:
+    """Run a command, echoing it, and raise on failure."""
+    print(f"  $ {' '.join(str(part) for part in cmd)}")
+    return subprocess.run(cmd, check=True, capture_output=True, text=True, **kwargs)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)